### PR TITLE
Fix results save location

### PR DESCRIPTION
**Pull request by yngtodd**
This addresses the issue of failing to write the genotype.json
when the directory does not exist. This is one of the simpler solutions,
and keeps things consistent with the defaul_model.txt of the advanced
example.

### CODE REVIEW

**Comment by yngtodd** [comment]
Fixes #65

File: common/darts/storage/genotype.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ def save_genotype(self, genotype: Genotype, filename='genotype.json') -> None:
             filename: name of the save file
         """
         genotype = self._replace_range(genotype)
+        os.makedirs(self.root, exist_ok=True)
         path = os.path.join(self.root, filename)
```
Comment (yngtodd) [inline common/darts/storage/genotype.py:27]: This sorts out the case where the save path does not already exist

**Comment by yngtodd** [comment]
haha Sorry, I misread that. Let me fix that real quick.

**Comment by jmohdyusof** [comment]
Tested, merged. Thanks!

**Pull request merged**